frontend/server/src/main/java/com/amazonaws/ml/mms/http/InvalidPluginException.java

```diff
@@ -0,0 +1,17 @@
+package com.amazonaws.ml.mms.http;
+
+public class InvalidPluginException extends RuntimeException {
```

Comment:
Not sure if this should be RuntimeException or not.

Reply:
I named it so because this occurs during runtime of MMS API loading. What is your suggestion? Should I change it to "InvalidStateException"?

Reply:
I'm thinking of make it a checked exception, that user must handle it.

Reply:
This plugin exception is not accessed by the user. This is thrown if there are issues while loading the plugin.